config/kafka-monitor.properties

```diff
@@ -39,6 +39,9 @@
     "zookeeper.connect": "localhost:2181",
     "bootstrap.servers": "localhost:9092",
     "produce.record.delay.ms": 100,
+    "produce.topic.autoTopicCreationEnabled": true,
```

Comment:
Yeah, they don't have to be changed in BasicEndToEndTest. I mentioned this only because in the previous patch they use different default value for autoTopicCreationEnabled.